dmaws/stacks.py

```diff
@@ -156,7 +156,7 @@ def create(self, create_dependencies=True):
 
         return True
 
-    def delete(self):
+    def delete(self, ignore_deps=False):
```

Comment:
`deps` could be both dependencies and dependants, it might be better to have the full word here, just like we do in `create(self, create_dependencies=True)`